fiducial_slam/nodes/fiducial_slam_node.py

```diff
@@ -408,6 +410,12 @@ def updatePose(self, tfs, imageTime):
                                                 quat, thisvar)
                 variance = v1
 
+        if self.medianFilterSamples > 0:
+            self.positionHistory.append(position)
+            if len(self.positionHistory) > self.medianFilterSamples:
+                self.positionHistory.pop(0)
+            position = numpy.median(self.positionHistory, axis=0)
```

Comment:
This will only take the median along one axis (x?), if we have multiple points we can median all three individually, and have the pose estimate be the three medians combined. I think that would provide a better estimate, would it not?